### ISSUE TITLE
EnableSCTenFootOverlayCheck label goes off the screen.

### ISSUE THREAD

**Issue by FlamingFox911** [DirtDiglett/Pressure-for-Steam#483]
Not sure how to make it multiline / wrap text, so I'll leave this here, in case anyone was interested.

![image](https://user-images.githubusercontent.com/11638944/38183129-f13a646c-360b-11e8-9867-a4d2c66e8424.png)

**Comment by Js41637** [comment]
Yeah I got no idea how to wrap checkboxes, as far as I know it's not possible.
